configs/WPLV2/config.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        WPLV2
+#define MANUFACTURER_ID   WRPL
+
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_GYRO_CLKIN
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#ifndef USE_OSD_HD
+#define USE_OSD_HD
+#endif
+
+#define USE_LED_STRIP
+
```

Comment:
LED_STRIP and OSD are user build options.
```suggestion
#define USE_BARO
```